### PR TITLE
Define the arguments of reduce’s callback function

### PR DESCRIPTION
**Pull request by robatron**
Hello, I'm a first-time user of clojure, and thought the [`reduce` core function docs](https://clojuredocs.org/clojure.core/reduce) could use clarification regarding its callback function's arguments.

I just noticed this project does not accept pull requests via github, but I figured this PR would be a good way to communicate my intended contribution before learning how to contribute a patch via JIRA.

### CODE REVIEW

File: src/clj/clojure/core.clj [hunk 1]
```diff
@@ -6684,15 +6684,15 @@
   [x] (instance? java.util.UUID x))
 
 (defn reduce
-  "f should be a function of 2 arguments. If val is not supplied,
```
Comment (robatron) [inline src/clj/clojure/core.clj]: Apologies for the hard wrap diff gore. I changed the first sentence from:

> f should be a function of 2 arguments.

to:

> f should be a function of 2 arguments: 1st, the accumulator, and 2nd the current element.

**Comment by AeroNotix** [comment]
You'll get the usual "We don't accept pull requests" comment. You need to use JIRA and file Article 239823 and fill in relevant release forms from Paragrah 63a Alpha, subparagraph 12 and ensure you click "accept trans dimensional JIRA brain uplink" when prompted.

**Comment by jafingerhut** [comment]
To be a little more precise :-)

You can find links for "Sign up for a JIRA account" and "Creating Tickets" on this page: http://dev.clojure.org/display/community/Contributing

Instructions for signing a Clojure Contributor Agreement are here, only necessary if you wish to contribute code to Clojure: https://clojure.org/community/contributing

**Comment by robatron** [comment]
@AeroNotix @jafingerhut indeed, hence:

> I just noticed this project does not accept pull requests via github, but I figured this PR would be a good way to communicate my intended contribution before learning how to contribute a patch via JIRA.

I.e., is the proposed change acceptable? As a complete clojure newb, I just want to make sure I'm not overlooking something obvious before contributing.

**Comment by AeroNotix** [comment]
^the above is exactly why a complicated contribution process is not helpful to beginners and drive-by pull requests.

**Comment by michaelklishin** [comment]
@robatron all it means that Clojure does not use GitHub pull requests. You will go through a hand crafted 16th century style software development process before you know if your change is worth accepting. They used to ask for a snail mail sent to North Carolina before you can contribute anything.

**Comment by jafingerhut** [comment]
@robatron There is only one person who can give the final answer on whether a contribution is acceptable or not, and that is Rich Hickey.  In practice, Alex Miller has a pretty good feel for what is interesting or not, and he reads the Clojure Google group where anyone can publish messages, and he often answers questions like this.  Or, create a JIRA ticket (no signing of a Clojure CA required), and he can address that question.

**Pull request closed without merging**